meta/documents/changelog_de.md

```diff
@@ -2,6 +2,9 @@
 
 ## v1.2.3 (2017-07-20)
 
+### Hinzugefügt
+- Für Parent-SKUs können unter den Plugineinstellungen in dem Tab "Parent-SKU" ein Pre- und Suffix angelegt werde. Diese werden dann automatisch gesetzt, sofern nicht schon eine ParentSKU existiert.
```

Comment:
Für Parent-SKUs können in den Plugin-Einstellungen im Tab "Parent-SKU" Pre- und Suffix angelegt werden. Diese werden automatisch gesetzt, sofern keine Parent-SKU existiert.